river/ensemble/adaptive_random_forest.py

```diff
@@ -372,6 +400,32 @@ class AdaptiveRandomForestClassifier(BaseForest, base.Classifier):
     nominal_attributes
         [*Tree parameter*] List of Nominal attributes. If empty, then assume that
         all attributes are numerical.
+    attribute_observer
+        [*Tree parameter*] The attribute observer (AO) algorithm used to monitor the class
```

Comment:
This description should be in the `notes`section (now being rendered in the documentation). You can add a note here: "See notes for more information regarding this attribute."

Reply:
Updated in commit [d247eb7](https://github.com/creme-ml/creme/pull/379/commits/d247eb7b873f17442e35e1b1df8054ad05611a4b).